modules/python-modules/syslogng/modules/kubernetes/scl/kubernetes.conf

```diff
@@ -78,6 +78,12 @@ block parser kubernetes-metadata-parser (prefix() template("$FILE_NAME")) {
                 template(`template`)
                 prefix(`prefix`)
             );
+            python(
+                class("syslogng.modules.kubernetes.KubernetesAPIEnrichment")
+                options(
+                    "prefix" => "`prefix`"
```

Comment:
We should publish the `in_pod` option too, I needed that in my deployment.

Reply:
good point. I am thinking about we can actually detect this value, instead of having to make this explicit. That would be a lot better.

I am thinking about trying to load the config file and falling back to the incluster config if the first one raises an exception.

@alltilla what do you think? I am trying in a moment if this would work.

Reply:
I am pushing another iteration where this is implemented, it seems there's even this function in the kubernetes python libs.